lib/plugins/milestones.js

```diff
@@ -0,0 +1,37 @@
+const Diffable = require('./diffable')
+
+module.exports = class Milestones extends Diffable {
+  find () {
+    let allMilestones = []
+
+    allMilestones.push(this.github.issues.getMilestones(Object.assign({}, {state: 'open'}, this.repo)).then(res => res.data))
+    allMilestones.push(this.github.issues.getMilestones(Object.assign({}, {state: 'closed'}, this.repo)).then(res => res.data))
+
+    return Promise.all(allMilestones)
+      .then(milestones => {
+        return milestones[0].concat(milestones[1])
+      })
```

Comment:
Switching to one API call should make this go away, but if it didn't, this could be cleaned up by calling `allMilestones.push(...newItems)`. The `...` will expand the array before pushing it onto the new list.